device/isis/routeradvanced.yaml

```diff
@@ -69,4 +69,11 @@ components:
           type: integer
           minimum: 1
           maximum: 60000
-          default: 5000  
+          default: 5000
+        enable_attached_bit:
+          description: >-
+            Attach bit indicates that the node is attached to another area 
+            or the Level 2 backbone. The purpose of an Attached-Bit is to accomplish Inter-Area Routing. 
+            When an L1/L2 router is connected to more than one area, it sets the Attached-bit on its L1 LSP.
+          type: boolean
```

Comment:
If the Attached bit is enabled , it indicates that the ISIS router ...  
Maybe add: This can cause a default route ( 0.0.0.0/0 ) to be installed by the receiving node. [  after confirming from spec ... but practically we see this as a visible side-effect on the DUT so might be good to call it out specifically ]

Reply:
modified